### PR TITLE
When installing on fedora dnf-utils required

### PR DESCRIPTION
**Pull request by galitf**
Signed-off-by: Galit <grosenth@redhat.com>

### CODE REVIEW

**Comment by galitf** [comment]
ci test please

**Comment by galitf** [comment]
ci test please

**Comment by galitf** [comment]
ci test please

**Comment by galitf** [comment]
ci test please

File: tests/unit/ovirtlago/test_runtest.py [hunk 1]
```diff
@@ -19,12 +19,15 @@ def dummy_test():
     return DummyTest(id=lambda: 123)
 
 
-@pytest.mark.parametrize('exc', [ExtractPathError, ExtractPathNoPathError])
+@pytest.mark.parametrize(
+    ('exc', 'remote_path'),
```
Comment (gbenhaim) [inline tests/unit/ovirtlago/test_runtest.py]: Are you sure that you can put tuple in here?
https://docs.pytest.org/en/latest/parametrize.html

File: tests/unit/ovirtlago/test_runtest.py [hunk 2]
```diff
@@ -19,12 +19,15 @@ def dummy_test():
     return DummyTest(id=lambda: 123)
 
 
-@pytest.mark.parametrize('exc', [ExtractPathError, ExtractPathNoPathError])
+@pytest.mark.parametrize(
```
Comment (gbenhaim) [inline tests/unit/ovirtlago/test_runtest.py]: I think that the commit message should explain why this change is needed, I guess it's because the interface of the exceptions was changed in lago.

File: tests/unit/ovirtlago/test_runtest.py (hunk 1 above)
Comment (galitf) [reply tests/unit/ovirtlago/test_runtest.py]: argnames – a comma-separated string denoting one or more argument names, or a list/tuple of argument strings.

**Comment by gbenhaim** [comment]
ci merge please
